tests/Phinx/Config/ConfigDefaultEnvironmentTest.php

```diff
@@ -37,12 +38,10 @@ public function testConfigReplacesTokensWithEnvVariables()
         $this->assertEquals('1234', $env['port']);
     }
 
-    /**
-     * @expectedException \RuntimeException
-     * @expectedExceptionMessage The environment configuration (read from $PHINX_ENVIRONMENT) for 'conf-test' is missing
-     */
     public function testGetDefaultEnvironmentOverridenByEnvButNotSet()
     {
+        $this->expectException(RuntimeException::class);
```

Comment:
As a good practice/code-sniff we should move the expect* methods directly before the part that is supposed to throw it.
If we dont do that, other code can have side effects with similar exceptions cloaking it.
So if we move it inline, we better use the best of this.

Also, please add an extra newline before and after the expect* method blocks for readability.

Reply:
I am actually writing a sniff for our company here (whitespace for readability), we could also add this one to the core sniffer repo if we want to.